pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Description
+*The intended change, motivation and consequences.*
+
+# Version
+[ ] I have increased the appropriate version number in https://github.com/chanzuckerberg/idseq-dag/blob/master/idseq_dag/__init__.py
+[ ] I have added release notes for my new version to https://github.com/chanzuckerberg/idseq-dag/blob/master/README.md
+
+Version numbers for this repo take the form X.Y.Z.
```

Comment:
the explanation is good but I don't think it belongs in the PR template because authors only need to read it once. I think including the same description in the README above is enough.

Reply:
ok I added it to https://github.com/chanzuckerberg/idseq-dag/blob/pr-template/README.md#release-notes and linked to that section from the PR template